script/npm_lint_plugins.js

```diff
@@ -1,20 +1,83 @@
 #!/usr/bin/env node
 /* eslint-disable no-var */
+/* eslint-disable no-console */
 
 var fs = require('fs');
 
-var childProcess = require('child_process');
-var { packageJsonDirs } = require('./plugin_webpack_directories');
+const { spawn, execSync } = require('child_process');
+const path = require('path');
+
+var { packageJsonDirsObject } = require('./plugin_webpack_directories');
+
+var dirs = packageJsonDirsObject();
+var dirsKeys = Object.keys(dirs);
+var passedArgs = process.argv.slice(2);
 
 function pluginDefinesLint(pluginPath) {
   var packageData = JSON.parse(fs.readFileSync(`${pluginPath}/package.json`));
 
   return packageData.scripts && packageData.scripts.lint;
 }
+if (passedArgs[0] && passedArgs[0][0] !== '-') {
```

Comment:
Please add an empty line after the method declaration and before the actual script